packages/applications-service-api/src/utils/stopwords.js

```diff
@@ -0,0 +1,258 @@
+const stopWords = {
+	english: [
+		'a',
+		'about',
+		'above',
+		'after',
+		'again',
+		'against',
+		'all',
+		'am',
+		'an',
+		'and',
+		'any',
+		'are',
+		"aren't",
+		'as',
+		'at',
+		'be',
+		'because',
+		'been',
+		'before',
+		'being',
+		'below',
+		'between',
+		'both',
+		'but',
+		'by',
+		"can't",
+		'cannot',
+		'could',
+		"couldn't",
+		'did',
+		"didn't",
+		'do',
+		'does',
+		"doesn't",
+		'doing',
+		"don't",
+		'down',
+		'during',
+		'each',
+		'few',
+		'for',
+		'from',
+		'further',
+		'had',
+		"hadn't",
+		'has',
+		"hasn't",
+		'have',
+		"haven't",
+		'having',
+		'he',
+		"he'd",
+		"he'll",
+		"he's",
+		'her',
+		'here',
+		"here's",
+		'hers',
+		'herself',
+		'him',
+		'himself',
+		'his',
+		'how',
+		"how's",
+		'i',
+		"i'd",
+		"i'll",
+		"i'm",
+		"i've",
+		'if',
+		'in',
+		'into',
+		'is',
+		"isn't",
+		'it',
+		"it's",
+		'its',
+		'itself',
+		"let's",
+		'me',
+		'more',
+		'most',
+		"mustn't",
+		'my',
+		'myself',
+		'no',
+		'nor',
+		'not',
+		'of',
+		'off',
+		'on',
+		'once',
+		'only',
+		'or',
+		'other',
+		'ought',
+		'our',
+		'ours',
+		'ourselves',
+		'out',
+		'over',
+		'own',
+		'same',
+		"shan't",
+		'she',
+		"she'd",
+		"she'll",
+		"she's",
+		'should',
+		"shouldn't",
+		'so',
+		'some',
+		'such',
+		'than',
+		'that',
+		"that's",
+		'the',
+		'their',
+		'theirs',
+		'them',
+		'themselves',
+		'then',
+		'there',
+		"there's",
+		'these',
+		'they',
+		"they'd",
+		"they'll",
+		"they're",
+		"they've",
+		'this',
+		'those',
+		'through',
+		'to',
+		'too',
+		'under',
+		'until',
+		'up',
+		'very',
+		'was',
+		"wasn't",
+		'we',
+		"we'd",
+		"we'll",
+		"we're",
+		"we've",
+		'were',
+		"weren't",
+		'what',
+		"what's",
+		'when',
+		"when's",
+		'where',
+		"where's",
+		'which',
+		'while',
+		'who',
+		"who's",
+		'whom',
+		'why',
+		"why's",
+		'with',
+		"won't",
+		'would',
+		"wouldn't",
+		'you',
+		"you'd",
+		"you'll",
+		"you're",
+		"you've",
+		'your',
+		'yours',
+		'yourself',
+		'yourselves'
+	],
+
+	welsh: [
```

Comment:
will these be used? i.e. is there a future ticket to include Welsh stopwords?

Reply:
They were in use but i decided to just go with English in the end.  I was keeping those in there as an example of Welsh stop-words.  I was in two mind whether to remove or not?

Reply:
maybe leave them in with a comment on the ticket to possibly re-include in future for searches when language set to Welsh?